### DISCUSSION TITLE
Favorites on Desktop

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#393] [Q&A] [answered]
In Cruiser Desktop, would it be possible to have several groups of favorites as we have in the APP?
Thanks.

**Comment by devemux86** [comment]
@ramondevic thanks for the suggestion.

Probably in the future, as currently the focus of development is the Android app. :slightly_smiling_face:

**Reply by ramondevic** [reply]
I have been using the Cruiser APP for a long time but I continue to use Kurviger WEB as Cruiser Desktop is very limited compared to Kurviger and I would like to fully work with the Cruiser platform.
I have the feeling that lately a lot of work has been done on the aesthetic part (and in my opinion a little too much unnecessary) of Cruiser APP and on the other hand the functional part of Cruiser Desktop has been completely abandoned, it's a shame.
Thanks.

(translated from Catalan to English)

**Comment by devemux86** [comment, marked as answer]
Desktop Cruiser started many years ago as a companion app used primarily by map / theme creators.
[OpenAndroMaps](https://www.openandromaps.org/en/), [Freizeitkarte](https://www.freizeitkarte-osm.de/android/en/index.html) and others use desktop Cruiser to create and test their maps / themes.

Currently only BRouter online works on desktop, so it isn't possible to use all Cruiser options there,
as it does not support all the parameters of Android BRouter offline service (internal or external).

---

On desktop it may be easier to use [BRouter website](https://brouter.de/brouter-web/) for route planning with Cruiser profile?
Export routes as gpx navigation tracks and import them **offline** in Cruiser:
- https://github.com/devemux86/cruiser/discussions/272

**Reply by ramondevic** [reply]
I am very interested in being able to have my favorite files and with BRouter it seems to be very precarious.

On the desktop I don't need the offline services as I usually work from home or somewhere stable with an internet connection.

Well, nothing, for now I'll continue with Kurviger Desktop, for me it's the best online route creation tool out there right now.

Let's see if some day not too far off it will be possible to have a Cruiser Desktop in good condition.

**Comment by MotoUKRider** [comment]
Going back to this - Could you add the icon "Grouping" setting to Desktop? (ie: so I can disable it)

It's frustrating seeing "5+" rather than a few extra icons, and sometimes they are further apart enough that grouping them into a single icon is a waste.

**Reply by devemux86** [reply]
@MotoUKRider

Do you mean the "Settings | Map | Appearance | Group icons until zoom" on Android?

Did we have this option on the desktop? I can add it, it should be easy. :slightly_smiling_face:

**Reply by devemux86** [reply]
It's more simple to disable the clustering on desktop because desktops are far more powerful than mobile devices.

**Reply by MotoUKRider** [reply]
@devemux86 Yes, that's the one - the grouping was sometimes spoiling a screenshot :confused: 
Thanks, that'd be really useful.

**Reply by devemux86** [reply]
@MotoUKRider 

Clustering is disabled on desktop in the latest:
- https://github.com/devemux86/cruiser/discussions/610
